Coordinator/Coordinator/Coordinators/MainCoordinator.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  MainCoordinator.swift
+//  Coordinator
+//
+//  Created by Анна Сычева on 28.12.2022.
+//
+
+import UIKit
+
+/// Координатор главного экрана
```

Comment:
```suggestion
/// Координатор главного флоу 
```
флоу целый а не 1 экран